tests/apollo/util/skvbc_history_tracker.py

```diff
@@ -26,20 +26,34 @@
 
 MAX_LOOKBACK=10
 
-def verify_linearizability(async_fn):
+
+def verify_linearizability(with_track=True):
```

Comment:
drop the 'with_track' parameter from the decorator, as it should comes from the decorated method...